build.js

```diff
@@ -23,4 +25,10 @@ rollup({
 	]
 }).then( function(bundle) {
 	bundle.write({ dest: 'dist/sofe.js', format: 'cjs' })
+	console.log("Wrote rollup bundle into dist")
+})
+.catch(function(err) {
```

Comment:
A fresh npm install resulted in being unable to build sofe. I had to upgrade all of the rollup dependencies to get everything working again. However, the build was not failing because there was no `.catch` here.